### DISCUSSION TITLE
Support configuring a JDBC driver per connection

### DISCUSSION THREAD

**Discussion by lukaseder** [dbeaver/dbeaver#15523] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

I often have to compare the behaviour of various in-memory database products *per version*, including H2, Derby, HSQLDB, SQLite, etc.

To switch versions, I have to change the global JDBC driver configuration, when it would be very convenient to configure the JDBC driver and version *per connection*.

**Describe the solution you'd like**

While I think that the current global driver setting is great, I'd love to be able to "fork" the setting per connection, not affecting the global driver setting or other connections. So, ideally, there should be some sort of extra button next to the existing button to edit the driver settings on a connection.

**Comment by flambchap** [comment, marked as answer]
Hi Lukas. You can do that already. To do that, open the Driver Manager (CRTL+3 -> Driver Manager), find the needed H2 driver, then click Copy and create a new driver based on the existing driver.

**Comment by lukaseder** [comment]
Thanks a lot for your help, @akilovich, that works for me. Also thanks for that awesome CTRL+3 trick. I wasn't aware of this. As a frequent Eclipse user of ~15 years, this will greatly help elsewhere, too! 👌
